packages/google-workspace-adapter/src/definitions/fetch/fetch.ts

```diff
@@ -0,0 +1,508 @@
+    },
+    element: {
+      topLevel: {
+        isTopLevel: true,
+        elemID: { parts: [{ fieldName: 'schemaName' }] },
+        serviceUrl: { path: 'https://admin.google.com/ac/customschema' },
+        alias: { aliasComponents: [{ fieldName: 'schemaName' }] },
+      },
+      fieldCustomizations: {
+        schemaId: {
+          hide: true,
+        },
+        fields: {
+          isMapWithDynamicType: true,
+        },
+      },
+    },
+  },
+  schema__fields: {
+    resource: {
+      directFetch: false,
+    },
+    element: {
+      fieldCustomizations: {
+        fieldId: {
+          hide: true,
+        },
+      },
+    },
+  },
+  building: {
+    requests: [
+      {
+        endpoint: {
+          path: 'https://admin.googleapis.com/admin/directory/v1/customer/my_customer/resources/buildings',
+        },
+        transformation: {
+          root: 'buildings',
+        },
+      },
+    ],
+    resource: {
+      directFetch: true,
+      serviceIDFields: ['buildingId'],
+    },
+    element: {
+      topLevel: {
+        isTopLevel: true,
+        elemID: { parts: [{ fieldName: 'buildingName' }] },
+        serviceUrl: { path: 'https://admin.google.com/ac/calendarresources/resources' },
+        alias: { aliasComponents: [{ fieldName: 'buildingName' }] },
+      },
+      fieldCustomizations: {
+        buildingId: {
+          hide: true,
+        },
+      },
+    },
+  },
+  room: {
+    requests: [
+      {
+        endpoint: {
+          path: 'https://admin.googleapis.com/admin/directory/v1/customer/my_customer/resources/calendars',
+        },
+        transformation: {
+          root: 'items',
+        },
+      },
+    ],
+    resource: {
+      directFetch: true,
+      serviceIDFields: ['resourceId'],
+    },
+    element: {
+      topLevel: {
+        isTopLevel: true,
+        elemID: { parts: [{ fieldName: 'resourceName' }] },
+        serviceUrl: { path: 'https://admin.google.com/ac/calendarresources/resources' },
+        alias: { aliasComponents: [{ fieldName: 'resourceName' }] },
+      },
+      fieldCustomizations: {
+        resourceId: {
+          hide: true,
+        },
+        generatedResourceName: {
+          omit: true,
+        },
+      },
+    },
+  },
+  feature: {
+    requests: [
+      {
+        endpoint: {
+          path: 'https://admin.googleapis.com/admin/directory/v1/customer/my_customer/resources/features',
+        },
+        transformation: {
+          root: 'features',
+        },
+      },
+    ],
+    resource: {
+      directFetch: true,
+      serviceIDFields: ['name'],
+    },
+    element: {
+      topLevel: {
+        isTopLevel: true,
+        serviceUrl: { path: 'https://admin.google.com/ac/calendarresources/resources' },
+        alias: { aliasComponents: [NAME_ID_FIELD] },
+      },
+    },
+  },
+})
+
+export const createFetchDefinitions = (
+  _fetchConfig: UserFetchConfig,
+): definitions.fetch.FetchApiDefinitions<Options> => ({
```

Comment:
nit: if you're not using the config you can also remove it
```suggestion
export const createFetchDefinitions = (): definitions.fetch.FetchApiDefinitions<Options> => ({
```
(and avoid passing the param from the adapter creator)